ts/features/itwallet/machine/eid/machine.ts

```diff
@@ -242,9 +244,11 @@ export const itwEidIssuanceMachine = setup({
           actions: "requestAssistance"
         },
         reset: {
-          target: "Idle"
+          target: "Idle" // TODO: handle navigation to an initial screen (TOS?)
```

Comment:
```suggestion
          target: "Idle"
```

This is not required. This event will never be dispatched, it works as a reset for testing purposes (Playground).